test/vsts.test.ts

```diff
@@ -37,3 +37,15 @@ test("Get file URL in VSTS with SSH", async () => {
     const link = await GitUrls["getUrlsCoreAsync"](configInfo);
     expect(link).toBe("https://ssh.visualstudio.com/Collection/_git/repo?path=%2Ftest%2Ffile&version=GBmaster&_a=contents");
 });
+
+test("Get selection block URL with column in VSTS", async () => {
+    const configInfo = {
+        remoteUrl: "https://vsts.visualstudio.com/Collection/_git/repo",
+        branchName: "master",
+        section: new Section(12, 23, 8, 9),
```

Comment:
Could better to be an interface intead of a class. I think this will be clear from API aspect:

```js
section: {
  startLine: 1,
  endLine: 2,
  startLineColumn: 11,
  endLineColumn: 12,
}
```